acceptance/config/asset_manager.go

```diff
@@ -152,6 +154,22 @@ func (a AssetManager) LifecycleDescriptor(kind ComboValue) builder.LifecycleDesc
 	return builder.LifecycleDescriptor{} // Unreachable
 }
 
+func (a AssetManager) LifecycleImage(kind ComboValue) string {
+	a.testObject.Helper()
+
+	switch kind {
+	case Current:
+		return a.lifecycleImage
+	case Previous:
```

Comment:
This is a bit weird. IDK, maybe it should always be respected. But it seems like if you went through the trouble to run the combo suite you should use the "real" one.

Reply:
If so, then we shouldn't be enabling the previous lifecycle option at all, but I think that there is value to ensure there is cross-compatibility.

Reply:
So I've been thinking about this some more, and I feel that no matter what we do, there will be some weirdness:

(a) Override lifecycle image in "previous" case - `LIFECYCLE_IMAGE=<some-image> make acceptance-all` produces weird results because the lifecycle used for analyze, restore, and export is ALWAYS "current"
(b) Don't override lifecycle image in "previous" case - `LIFECYCLE_IMAGE=<some-image> make acceptance-all` produces weird results because hey I provided the lifecycle image, why isn't it being used always???

As a lifecycle contributor I don't actually run `make acceptance-all`, so I can't really imagine falling into option (b). And option (a) just seems weirder and more likely to produce bugs.

I think whatever we decide to do, we should print a warning when `LIFECYCLE_IMAGE` is set for `make acceptance-all`. I could make that change.

Reply:
After thinking about this some more, I think the most intuitive solution would be to update this PR to include support for `PREVIOUS_LIFECYCLE_IMAGE`. Then we could just respect it if set and the combo is "previous".

Reply:
That makes sense. Did you update it yet?

Reply:
I'm so slow...

Reply:
Updated now.